### ISSUE TITLE
OPS-0012 English Composition Unit Opener page text overlap issue

### ISSUE THREAD

**Issue by srm-openstax** [openstax/cnx-recipes#4098]
We have done the coding as in the tagging legend on the Unit Opener page. But there is a issue with text overlap in the output file. Kindly check and resolve this issue.

![unit_opener_text_overlap](https://user-images.githubusercontent.com/86622862/132979469-247419cb-39a8-4863-b4cb-41ac449da0fa.png)
![Unit opener cnxml](https://user-images.githubusercontent.com/86622862/132979489-81213b4e-e283-4d5b-ab22-a467877d2205.png)

# Bug Location(s)
collection ID: col00001
server: https://copper-anteater-egyq5vxq.ws-us15.gitpod.io/
chapter/section: Unit 1 Opener page
surrounding text: Unit 1 is about you—who you are and what you bring to the

**Comment by wilkus27** [comment]
@lwtchu at this moment it is styled in the way where text **Introduction** always starts at the same place. It's done like that because unit title is not part of the page (pink part), it's just moved down. We can't change it because that would have an impact on webview. So what I did now, I just moved **Introduction** lower. This is how it looks:

![image](https://user-images.githubusercontent.com/40228854/135286278-cfcc247d-0cd7-4e67-b33a-fae6dd6d7b36.png)

But this is how it looks for unit with shorter title:

![image](https://user-images.githubusercontent.com/40228854/135286528-e1cb8eaa-43d5-4959-8326-07fce6c7cd0a.png)

Is it ok? Or maybe I should try to decrease font for unit titles, so the difference between long and short title will be less noticeable? Or maybe make pink part wider, so there will be more place for title and it won't take four lines? Let me know what do you think :slightly_smiling_face:

**Comment by lwtchu** [comment]
@wilkus27 can we increase the pink part by an inch? I'm not sure where it sits right now 🤔  maybe bump it up to 5.5 in wide or 5.25in wide

**Comment by wilkus27** [comment]
@lwtchu I did it but actually final effect is the same, because shorter title has now 2 lines, not 3:

![image](https://user-images.githubusercontent.com/40228854/135458415-49da01f0-72e3-4819-b45b-559e425f5dec.png)

Here is longer title:

![image](https://user-images.githubusercontent.com/40228854/135458501-73f9d2a1-a9d7-4a7c-8869-3af76b4e961c.png)

So I tried also put Introduction title just in the middle of the page, so there will be always large space between it and unit title:

![image](https://user-images.githubusercontent.com/40228854/135458795-730e3cd9-8a7c-4ddd-a339-defb8c48b223.png)

![image](https://user-images.githubusercontent.com/40228854/135458893-4141597a-d927-443f-b086-2e799ea810e0.png)

I don't know :thinking: Let me know what do you think.

**Comment by lwtchu** [comment]
@wilkus27 would it help to decrease the fontsize for the unit title to font-scale(6)? (down one)

**Comment by wilkus27** [comment]
@lwtchu This is how it looks:

![image](https://user-images.githubusercontent.com/40228854/135993189-fcfd9026-f6e8-4b52-9d8a-e405035fb849.png)

![image](https://user-images.githubusercontent.com/40228854/135993253-91b5841c-c82e-42c9-9ea1-f0e257b0da2e.png)

**Comment by lwtchu** [comment]
@wilkus27 in this new version is the pink part also widened (i also want to give a bit of breathing room for the intro text just in case

**Comment by wilkus27** [comment]
@lwtchu No, I decreased just font here. Let me try with wider page :wink:

**Comment by wilkus27** [comment]
@lwtchu Here is with wider pink part:

![image](https://user-images.githubusercontent.com/40228854/136059986-7fc3c3b1-eadc-444f-b14e-a2f85f070202.png)

![image](https://user-images.githubusercontent.com/40228854/136060055-d8ac1fda-2390-4129-97e3-8d851edb738a.png)
